apps/shared/y4m.c

```diff
@@ -7,10 +7,15 @@
 
 #include <assert.h>
 #include <inttypes.h>
+#include <limits.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
+#include "avif/avif.h"
+#include "avifutil.h"
```

Comment:
Optional: I remember IWYU allows foo.c to rely on the headers included by foo.h. This is why these two headers were not included. It's fine to follow the IWYU policy strictly.